templates/inspec/plural_resource.erb

```diff
@@ -14,4 +14,70 @@
 -%>
 <%= compile 'templates/license.erb' -%>
 
-<%= lines(autogen_notice :ruby) -%>
\ No newline at end of file
+<%= lines(autogen_notice :ruby) -%>
+
+class <%= object.name -%>s < Inspec.resource(1)
```

Comment:
I don't understand `Inspec.resource(1)` - can you explain?

Reply:
That's how you identify that a class is an InSpec resource (version 1). It allows InSpec to do some magic to hook these things up so they can be initialized via the `name` field set below, and ensures that the platform you declare you are running on matches the platform below.